lib/api/controllers/documentController.js

```diff
@@ -767,25 +768,31 @@ class DocumentController extends NativeController {
   async updateByQuery (request) {
     let query = request.getBodyObject('query');
     const changes = request.getBodyObject('changes');
+    const strict = request.getBoolean('strict');
     const silent = request.getBoolean('silent');
     const userId = request.getKuid();
     const refresh = request.getString('refresh', 'false');
     const source = request.getBoolean('source');
     const { index, collection } = request.getIndexAndCollection();
     const lang = request.getLangParam();
+    const methodName = 'core:storage:public:document:updateByQuery';
```

Comment:
To avoid duplication of string in the ask (line `785`) and throw of error (line `793`). Everything is inspired from the function `_mChange` (line `838`) which take an argument `methodName`

Reply:
Yes indeed, I did not have its use in the error my bad